client/src/components/SelectDropdown/SelectDropdown.css

```diff
@@ -0,0 +1,12 @@
+.selectDropdown {
+  width: 184px;
+  height: 40px;
```

Comment:
Rather than giving these an explicit height and width, can you give these padding to give the appropriate ratio of spacing?